tests/data/firefox/tinderbox-builds/mozilla-central-linux/1374583609/README.md

```diff
@@ -0,0 +1,3 @@
+This is supposed to be an empty directory to serve as a test case for a missing binary.
+
+See Issue #143 "TinderboxScraper: Omit folders without binaries"
```

Comment:
But that is using latest build. So whenever someone of us will add more entries to this folder, this specific test will be invalid. Or does this behavior only work for the request to get the latest build?